.github/workflows/main.yml

```diff
@@ -40,6 +40,18 @@ jobs:
         run: git fetch
       - name: Lint git commit messages
         run: make check-gitlint
+  test:
+    needs: [build, check]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v5
+        with:
+          go-version: "stable"
+      - uses: actions/checkout@v4
+      - name: Install build tools
+        run: make build-tools
```

Comment:
From the Makefile it looks like `build-tools` target is not a requirement for `test`.

Reply:
Fixed